### ISSUE TITLE
"a little" vs "a little bit" for describing induced charge

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#378]
Sometimes when describing movement of charges in the wall, the design document uses "a little", other times it uses "a little bit". @terracoda can we use one everywhere instead of both?

**Comment by jessegreenberg** [comment]
Currently, implementation uses "a little bit" everywhere.

**Comment by terracoda** [comment]
@jessegreenberg, I apologize for the inconsistencies in the design documents. 

The induced charge descriptions when a balloon is actually at the wall, should always be 1 of three options:
1. {{a little bit}}
2. {{a lot}}
3. {{quite a lot}}

These three parameters show up in several places grab, rubbing at wall, wall's dynamic description. I can't think of any other places at the moment.

The only place that "a little" should show up by itself is in the dragging induced charge descriptions, and the similar jumping induced charge alerts.

Maybe we can remove "a little" from the dragging/jumping alerts? 

Let me review this idea and get back to you asap.

**Comment by terracoda** [comment]
@jessegreenberg, I think this confusion was caused by my own inconsistency in the Two-balloon design document.

All the examples there that describe a negatively charged balloon at the wall with a small induced charge effect should be described as "a little bit" and not as "a little".

Good catch! I have corrected the errors I the design document.

As I mentioned above, the other place where "a little" is used is in the Approach and Return descriptions of induced charge when dragging right and left. I am assuming these are fine as is:
**Approaching wall:**
- Negative charges in wall [[begin to move away]] from yellow balloon.
- Negative charges in wall [[move away a little more]] from yellow balloon. _(Note that there is no need to split the move-away phrase as I originally requested in the design document. I don't remember where we landed with this decision.)_

**Moving a way from  wall:**
- Negative charges in wall [[begin to return]].
- Negative charges in wall [[return a little more]].

**Comment by terracoda** [comment]
@jessegreenberg, assigning back to you.

**Comment by terracoda** [comment]
And assuming no changes needed for the Approach and Return dragging alerts. We will start a separate issue if needed for those.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, https://github.com/phetsims/balloons-and-static-electricity/issues/378#issuecomment-377992543 is how current version is behaving, so I think we can close. Thanks for clarifying.
